src/SSHDebugPS/UI/ContainerInstance.cs

```diff
@@ -15,31 +15,22 @@
 
 namespace Microsoft.SSHDebugPS.Docker
 {
-    public interface IContainerInstance : IEquatable<IContainerInstance>, INotifyPropertyChanged
+    public interface IContainerInstance : IEquatable<IContainerInstance>
```

Comment:
>public [](start = 4, length = 6)

Should these classes still be public?

Reply:
Yes, because WPF is awesome and needs it to be. If its internal, they won't load.